src/components/ScholarshipForm.jsx

```diff
@@ -44,11 +44,12 @@ function ScholarshipForm({ scholarship }) {
   const navigate = useNavigate();
 
   const isDesktop = useMediaQuery((theme) => theme.breakpoints.up('md'));
-  const { t } = useTranslation(['scholarshipForm', 'common']);
+  const { t } = useTranslation(['scholarshipForm', 'common', 'validation']);
+  const { t: validationT } = useTranslation('validation');
 
   const formik = useFormik({
     initialValues: scholarship.data,
-    validationSchema,
+    validationSchema: validationSchema(validationT),
```

Comment:
So I think, even if the errors don't change immediately, this _does_ affect future errors. We can probably just add a listener on the language change that looks like this:
```js
i18n.on("languageChanged", () => formik.errors.length && formik.validateForm());
```

Reply:
hmm so it works, but it takes a while to re-load.

Reply:
Another option would be to clear `formik.errors` when the translation change happens so we don't need to do this.

Reply:
So basically:
```js
i18n.on("languageChanged", () => formik.setErrors({}));
```